client-default/src/test/scala/com.gu.contentapi.client/GuardianContentClientTest.scala

```diff
@@ -37,10 +30,6 @@ class GuardianContentClientTest extends FlatSpec with Matchers with ScalaFutures
 
   implicit override val patienceConfig = PatienceConfig(timeout = Span(5, Seconds))
 
-  "client interface" should "be using a correctly configured backoff strategy" in {
```

Comment:
useless test.

Reply:
It was testing that the backoff strategy sent to the client was being applied as expected

Reply:
Unlike the other test I deleted, I think this test is a valid test as it is testing the backoff strategy value. 
Although, as I mentioned in my previous comment, we should try to assert the effect of those values instead of the values themselves.

Reply:
Another reason why I deleted this test is that the backoff strategy is not a part of `GuardianContentClient` anymore. The backoff strategy is tested in `GuardianContentClientBackoffTest`